ConsoleView.h

```diff
@@ -17,32 +18,31 @@ class ConsoleView
 {
 
 public:
-    ConsoleView(){};
-  
-    
-    
-    
     // PrintClass Name will be used to Print the names of the class being passed by the user
     //@ Para: UMLClass object 
-    void PrintClassName(UMLClass X){}
+    void Print_Class_Name(UMLClass X);
     
     //PrintClass Attribute will Print a List of Attributes which are stored in the specified class
     //@Para: UMLClass Object  
-    void PrintClassAttribute(UMLClass X){}
+    void Print_Class_Attribute(UMLClass X);
     
     //PrintClassRelations will Print the current list of Relationships the class being passed has
     //@para: UmlClass Obj
-    void PrintClassRelations(UMLRelationship X){}
+    void Print_Class_Relations(UMLRelationship X);
     
     //PrintListofClass will Print out the list of classes currently in the Model
     //@para: list<string> 
-    void PrintAllClasses(std::list<std::string> x){}
+    void Print_All_Classes(std::list<std::string> x);
     
     
     //Print The current Soruce and Dest of the Relationship 
     //@para: UMLRelationship obj
-    void PrintSourceandDest(UMLRelationship x){}
-    
+    void Print_Source_and_Dest(UMLRelationship x);
+
+    //Print All the classes stored in the model
+    //@para: UMLMODEL
+    void Print_All_Classes(UMLModel x);
```

Comment:
The view would never have a reason to receive the UMLModel. If you're overloading this with the print_all_classes method from above that takes a list of string, you should make this one take in a list of classes instead, that way we can either print all classes from a UMLClass list or a string list

Reply:
Yes, thats exactly what'll be happening. The controller will call the model to actually get the list of strings or list of UMLClass objects and then pass it to the view. The view is solely responsible for displaying that